internal/deviations/deviations.go

```diff
@@ -269,6 +274,8 @@ var (
 
 	p4rtMissingDelete = flag.Bool("deviation_p4rt_missing_delete", false, "Device does not support delete mode in P4RT write requests")
 
+	P4rtUnsetElectionIDUnsupported = flag.Bool("deviation_p4rt_unsetelectionid_unsupported", false, "Device closes stream channel for P4RT unset election ID")
```

Comment:
Please unexport this variable i.e set variable name as `p4rtUnsetElectionIDUnsupported`

Reply:
Thanks, this is taken care. Please check.